openapi3.yaml

```diff
@@ -208,6 +220,12 @@ paths:
       tags:
         - Mosaic
       summary: update mosaic layers
+      parameters:
```

Comment:
add to description (above all other layers)

Reply:
fixed in 5ea5f76